book.go

```diff
@@ -587,12 +595,24 @@ func (bk *book) applyOptions(opts ...Option) error {
 	for _, opt := range opts {
 		_ = opt(nil)
 	}
-	opts = setupBuiltinFunctions(opts...)
+	if err := bk.applyBuiltinFunctions(); err != nil {
+		return err
+	}
 	for _, opt := range opts {
 		if err := opt(bk); err != nil {
 			return err
 		}
 	}
+	// bk.path is required for builtin.File.
+	if _, ok := bk.funcs["file"]; !ok {
+		root, err := bk.generateOperatorRoot()
```

Comment:
[Q] Will the evaluation in the included book be relative to the path to include?

Reply:
@k2tzumi 
Fixed https://github.com/k1LoW/runn/pull/1234/commits/c4c74d3bf75e9a36fe53fd0ffa2dcb8c54a7a6b7

Thank you!!